test/e2e/scenario/microfrontend.scenario.ts

```diff
@@ -126,8 +139,69 @@ describe('microfrontend', () => {
       expect(event?.context?.handlingStack).toMatch(HANDLING_STACK_REGEX)
     })
 
+  describe('console apis', () => {
+    const consoleApis = ['log', 'debug', 'error', 'info', 'warn'] as const
+
+    consoleApis.forEach((api) => {
+      createTest(`expose handling stack for console.${api}`)
+        .withLogs(LOGS_CONFIG)
+        .withLogsInit((configuration, api: (typeof consoleApis)[number]) => {
+          window.DD_LOGS!.init(configuration)
+
+          function testHandlingStack() {
+            console[api]('foo')
+          }
+
+          testHandlingStack()
+        }, api)
+        .run(async ({ intakeRegistry }) => {
+          await flushEvents()
+
+          const event = intakeRegistry.logsEvents[0]
+
+          await flushBrowserLogs()
+
+          expect(event).toBeTruthy()
+          expect(event?.context).toEqual({
+            handlingStack: jasmine.stringMatching(HANDLING_STACK_REGEX),
+          })
+        })
+    })
+  })
+
+  describe('logger apis', () => {
+    const loggerApis = ['log', 'ok', 'debug', 'info', 'notice', 'warn', 'error', 'critical', 'alert', 'emerg'] as const
```

Comment:
**💬 suggestion:** ‏It takes more than 1 minute to run all microfrontends e2e tests locally. I don't think it's worth covering every log level, and using a single one of them should be sufficient. The code is the same for all of them anyway, and unit tests should have the full coverage.